silverback/main.py

```diff
@@ -297,7 +297,7 @@ def broker_task_decorator(
         ):
             raise ContainerTypeMismatchError(task_type, container)
 
-        elif container and isinstance(container, ContractEventWrapper):
```

Comment:
@antazoey this one little accident calls it to poll the entire of history, causing massive spend on `eth_getLogs`

Reply:
oh jeez, ya that isnt good, is there a bug report in core?

Reply:
didn't make one yet but was going to (unless you beat me to it)

not sure how you override this though, `__bool__`?

Reply:
yes, exactly, `__bool__`